status.sh

```
@@ -0,0 +1,151 @@
+        done
+    else
+        echo "   • No log directory found"
+    fi
+}
+
+# Функция для получения информации о базе данных
+get_database_info() {
+    print_colored "🗄️ Database:" "$BLUE"
+    
+    if [ -f "bot_database.db" ]; then
+        local size=$(du -h "bot_database.db" | cut -f1)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)